scripts/pipelines/acquia-pipelines.yml

```diff
@@ -16,6 +18,8 @@ events:
               - composer validate --no-check-all --ansi
               - composer install --ansi
               - source ${BLT_DIR}/scripts/pipelines/setup_env
+              # - nvm install v6.9.1
```

Comment:
This seems like something better executed in your front end script.

Reply:
@grasmash my concern with that is that the local already has these things installed, where as the pipelines still does not. The examples I've seen from several other projects are doing a similar method to ^^. Not sure that everyone "will" but I couldn't find any good documentation on how to do it, so was trying to provide some basic examples.